clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -237,22 +228,25 @@ object UncommittedGarbageCollector {
     cachedAddresses.write.text(s"$reportDst/deleted.text")
   }
 
-  private def reportPath(storageNamespace: String, runID: String): String = {
+  private def runPath(storageNamespace: String, runID: String): String = {
```

Comment:
Can we name it 'formatRunPath' or something that we don't think we are actually running something.